packages/platform/types/teams/outputs/team.output.ts

```diff
@@ -0,0 +1,128 @@
+import { ApiProperty as DocsProperty, ApiPropertyOptional } from "@nestjs/swagger";
+import { Expose } from "class-transformer";
+import { IsBoolean, IsInt, IsOptional, IsString, IsUrl, Length } from "class-validator";
+
+export class TeamOutputDto {
+  @IsInt()
+  @Expose()
+  @DocsProperty()
+  readonly id!: number;
+
+  @IsInt()
+  @IsOptional()
+  @Expose()
+  readonly parentId?: number;
```

Comment:
missing ApiPropertyOptional

Reply:
Well spotted - fixed in [fix: team parentId docs](https://github.com/calcom/cal.com/pull/18019/commits/42da64495858e4191734dde3036f146a90fb5695)